docs/source/index.rst

```diff
@@ -133,78 +128,27 @@ Extract stage output might look like this:
 Transform Stage
 ---------------
 
-The transform stage converts the clean and standardized data into a form that
-closely resembles what is expected by the target database or service.
-
-These are the main steps performed by transform stage:
-
-1. **Merge Extract Stage tables into unified form**
-
-    If in guided transform mode, the Transform Stage will apply the
-    user-defined transform function. This function specifies how the individual
-    tables from the Extract Stage should be merged into a single table.
-
-2. **Create unique identifiers for concepts**
-
-    For most concepts, the contributor of the data provides their own IDs to
-    uniquely identify the concept within a dataset. But for some, such as
-    diagnosis and phenotype observation events, an event ID isn't typically
-    provided.
-
-    The Transform Stage will create its own unique identifiers and assign one
-    to every concept instance. Most unique identifiers just equate to the ID
-    provided by the data contributor, while others are created from a
-    combination of values (e.g. diagnosis event = participant ID + diagnosis
-    name + participant age at event).
-
-3. **Convert from standard form to target form**
-
-    Next, the Transform Stage emits target entity instance dicts.
-
-    Properties in the unified form (standard concept attributes) are converted
-    to target concept properties using the Kids First target API configuration
-    file.
-
-4. **Apply common cleaning for the target service**
-
-    This includes actions like filling in leftover null or unknown values with
-    standard values (Not Reported, Unknown, etc).
-
-
-Transform stage output might look like this:
-
-.. code-block:: json
-
-    {
-        "participant": [
-            {
-                "endpoint": "/participants",
-                "id": "P001",
-                "links": {
-                    "family_id": "f1",
-                    "study_id": null
-                },
-                "properties": {
-                    "affected_status": "Not Reported",
-                    "consent_type": "GRU",
-                    "ethnicity": "Not Reported",
-                    "external_id": "1",
-                    "gender": "Female",
-                    "is_proband": "True",
-                    "race": "Not Reported",
-                    "visible": "Not Reported"
-                }
-            }
-        ]
-    }
-
-
+The transform stage applies the user-defined transform function which merges
+the individual tables created by the Extract Stage into one or more "records"
+tables that each contain all of the descriptive information for an instance of
+a real world entity or event on each row.
```

Comment:
This new statement looks enough sufficient for me. I wonder if it's enough informative for newcomers though.